test/parallel/test-util-inspect.js

```diff
@@ -2723,3 +2723,21 @@ assert.strictEqual(
       '\x1B[2mdef: \x1B[33m5\x1B[39m\x1B[22m }'
   );
 }
+
+// Test changing util.inspect.colors colors and aliases. Since this might have
+// side effects, we do it last.
+{
```

Comment:
Given possible side effects would it be better to wrap this in a child process?

Reply:
I suggest to clone the original value and to reset it at the end.

Reply:
A third possibility is to put it in its own test file.

Reply:
I went with saving and restoring the original value. PTAL